src/components/inputs/creatable-select-input/creatable-select-input.js

```diff
@@ -135,25 +181,37 @@ export class CreatableSelectInput extends React.Component {
                   }
                 : undefined
             }
+            onChange={(value, info) =>
+              // selectedOptions is either an array, or a single option
+              // depending on whether we're in multi-mode or not (isMulti)
+              this.props.onChange(
+                {
+                  target: { name: this.props.name, value },
+                  persist: () => {},
+                },
+                info
+              )
+            }
+            onFocus={this.props.onFocus}
+            onInputChange={this.props.onInputChange}
+            options={this.props.options}
+            placeholder={this.props.placeholder}
+            tabIndex={this.props.tabIndex}
+            tabSelectsValue={this.props.tabSelectsValue}
+            value={this.props.value}
+            // Creatable props
+            allowCreateWhileLoading={this.props.allowCreateWhileLoading}
             formatCreateLabel={
               this.props.formatCreateLabel ||
               (inputValue =>
                 this.props.intl.formatMessage(messages.createLabel, {
                   inputValue,
                 }))
             }
-            noOptionsMessage={
-              this.props.noOptionsMessage ||
-              (({ inputValue }) =>
-                inputValue === ''
-                  ? this.props.intl.formatMessage(
-                      messages.noOptionsMessageWithoutInputValue
-                    )
-                  : this.props.intl.formatMessage(
-                      messages.noOptionsMessageWithInputValue,
-                      { inputValue }
-                    ))
-            }
+            isValidNewOption={this.props.isValidNewOption}
```

Comment:
isNewOptionValid?

Reply:
Those props are based on the `react-select` props and I would not want to deviate from them to avoid confusion.

See https://react-select.com/props#creatable-props